namespaces/Representor.md

```diff
@@ -0,0 +1,107 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Object Element)
+
+The Resource represents a Hypermedia Resource, with its available relations, transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `relations` - (array[Relation]) - The relations from this resource.
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - The embedded relations (resources) from this resource.
+    - `relation` - (string) - An optional relation to this resource, when embedded in another resource.
+- `content` (object) - Any properties of the resource.
+
+## Relation (Object Element)
+
+A relation element representing a relation to another resource.
```

Comment:
Relation really blurs with with transition. Is this the "Siren" approach (or my ignorance of hypermedia project' charter). Do we define `GET` transitions as "relation" whereas anything else is just "transition"? cc @fosrias @smizell